### YamlMime:ManagedReference
items:
  - uid: com.microsoft.azure.eventhubs._event_hub_exception
    id: _event_hub_exception
    parent: com.microsoft.azure.eventhubs
    children:
      - 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final String)'
      - 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final String,final Throwable)'
      - 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final Throwable)'
      - com.microsoft.azure.eventhubs._event_hub_exception.getContext()
      - com.microsoft.azure.eventhubs._event_hub_exception.getIsTransient()
      - com.microsoft.azure.eventhubs._event_hub_exception.getMessage()
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: EventHubException
    nameWithType: EventHubException
    fullName: com.microsoft.azure.eventhubs.EventHubException
    type: Class
    source:
      remote: &ref_0
        path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
        branch: master
        repo: 'https://github.com/Azure/azure-event-hubs-java'
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 11
    package: com.microsoft.azure.eventhubs
    summary: '<p>Dies ist die Basis-Ausnahme, die für alle Fehlerfälle Servicebus Sequenzelement erzeugt. </p>'
    syntax:
      content: public class EventHubException
    inheritance:
      - java.lang.Object
      - Exception
  - uid: 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final String)'
    id: 'EventHubException(final boolean,final String)'
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: 'EventHubException(final boolean isTransient, final String message)'
    nameWithType: 'EventHubException.EventHubException(final boolean isTransient, final String message)'
    fullName: 'com.microsoft.azure.eventhubs.EventHubException.EventHubException(final boolean isTransient, final String message)'
    overload: com.microsoft.azure.eventhubs._event_hub_exception.EventHubException*
    type: Constructor
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 24
    package: com.microsoft.azure.eventhubs
    syntax:
      content: 'public EventHubException(final boolean isTransient, final String message)'
      parameters:
        - id: isTransient
          type: 4dda0764
        - id: message
          type: 3d93591e
  - uid: 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final String,final Throwable)'
    id: 'EventHubException(final boolean,final String,final Throwable)'
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: 'EventHubException(final boolean isTransient, final String message, final Throwable cause)'
    nameWithType: 'EventHubException.EventHubException(final boolean isTransient, final String message, final Throwable cause)'
    fullName: 'com.microsoft.azure.eventhubs.EventHubException.EventHubException(final boolean isTransient, final String message, final Throwable cause)'
    overload: com.microsoft.azure.eventhubs._event_hub_exception.EventHubException*
    type: Constructor
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 36
    package: com.microsoft.azure.eventhubs
    syntax:
      content: 'public EventHubException(final boolean isTransient, final String message, final Throwable cause)'
      parameters:
        - id: isTransient
          type: 4dda0764
        - id: message
          type: 3d93591e
        - id: cause
          type: fa00b19f
  - uid: 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final Throwable)'
    id: 'EventHubException(final boolean,final Throwable)'
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: 'EventHubException(final boolean isTransient, final Throwable cause)'
    nameWithType: 'EventHubException.EventHubException(final boolean isTransient, final Throwable cause)'
    fullName: 'com.microsoft.azure.eventhubs.EventHubException.EventHubException(final boolean isTransient, final Throwable cause)'
    overload: com.microsoft.azure.eventhubs._event_hub_exception.EventHubException*
    type: Constructor
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 30
    package: com.microsoft.azure.eventhubs
    syntax:
      content: 'public EventHubException(final boolean isTransient, final Throwable cause)'
      parameters:
        - id: isTransient
          type: 4dda0764
        - id: cause
          type: fa00b19f
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getContext()
    id: getContext()
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: getContext()
    nameWithType: EventHubException.getContext()
    fullName: ErrorContext com.microsoft.azure.eventhubs.EventHubException.getContext()
    overload: com.microsoft.azure.eventhubs._event_hub_exception.getContext*
    type: Method
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 62
    package: com.microsoft.azure.eventhubs
    syntax:
      content: public ErrorContext getContext()
      return:
        type: com.microsoft.azure.eventhubs._error_context
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getIsTransient()
    id: getIsTransient()
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: getIsTransient()
    nameWithType: EventHubException.getIsTransient()
    fullName: boolean com.microsoft.azure.eventhubs.EventHubException.getIsTransient()
    overload: com.microsoft.azure.eventhubs._event_hub_exception.getIsTransient*
    type: Method
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 57
    package: com.microsoft.azure.eventhubs
    summary: '<p>Ein boolescher Wert, der angibt, ob die Ausnahme ein vorübergehender Fehler ist. </p>'
    syntax:
      content: public boolean getIsTransient()
      return:
        type: 4fc6e284
        description: '<p>Gibt "true" zurück, wenn der Benutzer den Vorgang wiederholen kann, der die Ausnahme ohne weiteren Eingriff generiert hat. </p>'
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getMessage()
    id: getMessage()
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: getMessage()
    nameWithType: EventHubException.getMessage()
    fullName: String com.microsoft.azure.eventhubs.EventHubException.getMessage()
    overload: com.microsoft.azure.eventhubs._event_hub_exception.getMessage*
    type: Method
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 43
    package: com.microsoft.azure.eventhubs
    syntax:
      content: public String getMessage()
      return:
        type: '26831127'
references:
  - uid: 4dda0764
    spec.java:
      - name: final boolean
        fullName: final boolean
  - uid: 3d93591e
    spec.java:
      - name: final String
        fullName: final String
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.EventHubException*
    name: EventHubException
    nameWithType: EventHubException.EventHubException
    fullName: com.microsoft.azure.eventhubs.EventHubException.EventHubException
    package: com.microsoft.azure.eventhubs
  - uid: fa00b19f
    spec.java:
      - name: final Throwable
        fullName: final Throwable
  - uid: '26831127'
    spec.java:
      - name: String
        fullName: String
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getMessage*
    name: getMessage
    nameWithType: EventHubException.getMessage
    fullName: String com.microsoft.azure.eventhubs.EventHubException.getMessage
    package: com.microsoft.azure.eventhubs
  - uid: 4fc6e284
    spec.java:
      - name: boolean
        fullName: boolean
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getIsTransient*
    name: getIsTransient
    nameWithType: EventHubException.getIsTransient
    fullName: boolean com.microsoft.azure.eventhubs.EventHubException.getIsTransient
    package: com.microsoft.azure.eventhubs
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getContext*
    name: getContext
    nameWithType: EventHubException.getContext
    fullName: ErrorContext com.microsoft.azure.eventhubs.EventHubException.getContext
    package: com.microsoft.azure.eventhubs
  - uid: com.microsoft.azure.eventhubs._authorization_failed_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._authorization_failed_exception.yml
    name: AuthorizationFailedException
    nameWithType: AuthorizationFailedException
    fullName: com.microsoft.azure.eventhubs.AuthorizationFailedException
    type: Class
    summary: |-
      <p>Autorisierung Ausnahmefehler ausgelöst wird, wenn Fehler aufgetreten ist, während die Benutzerberechtigungen zum Ausführen der gewünschten Vorgänge autorisieren. Wenn diese Ausnahme festgestellt Benutzer sollte überprüfen Sie, ob die Token-Schlüssel in der Verbindungszeichenfolge angegeben (z. B. mit dem übergebenen <xref uid="com.microsoft.azure.eventhubs._event_hub_client.createFromConnectionString(final String)" data-throw-if-not-resolved="false">com.microsoft.azure.eventhubs.EventHubClient#createFromConnectionString(String)</xref>) gültig ist, und verfügt über die korrekte Ausführung Recht für die vorgesehenen Vorgänge (z. B. empfangsaufruf Anspruch zugeordnete die Schlüssel bzw. das Token Lauschen müssen).</p>
      <p></p>
    syntax:
      content: public class AuthorizationFailedException extends EventHubException
  - uid: com.microsoft.azure.eventhubs._communication_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._communication_exception.yml
    name: CommunicationException
    nameWithType: CommunicationException
    fullName: com.microsoft.azure.eventhubs.CommunicationException
    type: Class
    summary: |-
      <p>Diese Ausnahme wird ausgelöst, wenn ein Client-Side-Konnektivitätsproblem vorhanden ist. Konnektivität der Clienteinstellungen für den Dienst empfängt diese Ausnahme Benutzer wann überprüft werden soll:<itemizedlist><listitem><p>Überprüfen Sie die richtige Hostname und Portnummer, die im Endpunkt verwendet. </p></listitem><listitem><p>Überprüfen Sie für alle möglichen Proxyeinstellungen, die Amqp-Ports zu blockieren, kann </p></listitem><listitem><p>Überprüfen Sie alle Firewalleinstellungen, die Amqp Ports blockieren können </p></listitem><listitem><p>Überprüfen Sie für jede allgemeine Probleme bei der Netzwerkverbindung sowie die Netzwerklatenz. </p></listitem></itemizedlist></p>
      <p></p>
    syntax:
      content: public class CommunicationException extends EventHubException
  - uid: com.microsoft.azure.eventhubs._illegal_entity_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._illegal_entity_exception.yml
    name: IllegalEntityException
    nameWithType: IllegalEntityException
    fullName: com.microsoft.azure.eventhubs.IllegalEntityException
    type: Class
    summary: |-
      <p>Diese Ausnahme tritt aus den folgenden Gründen:<itemizedlist><listitem><p>Wenn die entitätsbenutzer versucht, eine Verbindung herzustellen, ist nicht vorhanden </p></listitem><listitem><p>Die Entität Benutzer möchte die Verbindung ist deaktiviert. </p></listitem></itemizedlist></p>
      <p></p>
    syntax:
      content: public class IllegalEntityException
  - uid: com.microsoft.azure.eventhubs._operation_cancelled_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._operation_cancelled_exception.yml
    name: OperationCancelledException
    nameWithType: OperationCancelledException
    fullName: com.microsoft.azure.eventhubs.OperationCancelledException
    type: Class
    summary: '<p>Diese Ausnahme wird ausgelöst, wenn die zugrunde liegenden Amqp-Ebene treffen, einen nicht ordnungsgemäßen Link abbrechen oder Trennen der Verbindung auf unerwartete Weise. </p>'
    syntax:
      content: public class OperationCancelledException
  - uid: com.microsoft.azure.eventhubs._payload_size_exceeded_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._payload_size_exceeded_exception.yml
    name: PayloadSizeExceededException
    nameWithType: PayloadSizeExceededException
    fullName: com.microsoft.azure.eventhubs.PayloadSizeExceededException
    type: Class
    summary: |-
      <p>Diese Ausnahme wird ausgelöst, wenn der Benutzer versucht, Daten oder vermittelte Nachrichten, die die zulässige Nutzlastgröße gemäß der Definition durch den Dienst Zustellversuchen senden. Beachten Sie, die in einem Batch Szenario den Grenzwert senden kann mögliche Batch Aufwand einschließen.</p>
      <p></p>
    syntax:
      content: public class PayloadSizeExceededException
  - uid: com.microsoft.azure.eventhubs._quota_exceeded_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._quota_exceeded_exception.yml
    name: QuotaExceededException
    nameWithType: QuotaExceededException
    fullName: com.microsoft.azure.eventhubs.QuotaExceededException
    type: Class
    syntax:
      content: public class QuotaExceededException
  - uid: com.microsoft.azure.eventhubs._receiver_disconnected_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._receiver_disconnected_exception.yml
    name: ReceiverDisconnectedException
    nameWithType: ReceiverDisconnectedException
    fullName: com.microsoft.azure.eventhubs.ReceiverDisconnectedException
    type: Class
    summary: |-
      <p>Diese Ausnahme wird ausgelöst, wenn eine EventHubReceiver aufgrund einer der folgenden Gründe getrennt wird:<itemizedlist><listitem><p>Benutzer versucht, einen Empfänger nicht Epoche an einen Event Hub-Partition verbinden, wenn es ein Empfänger Epoche ist verbunden, der der Partition. </p></listitem><listitem><p>Verwenden Sie einen Empfänger Epoche für eine bestimmte Partition jedoch ein anderer Epoche Empfänger mit einem höheren Wert für den Beginn der Epoche eine Verbindung mit derselben Partition her. </p></listitem></itemizedlist>Benutzer sollten entweder der gesamte Code nicht Epoche Empfänger verwenden, oder stellen Sie sicher, dass es ist nur eine Epoche Empfänger, die Verarbeitung einer bestimmten Partitions zu einem bestimmten Zeitpunkt zeitlich sicherstellen.</p>
      <p></p>
    syntax:
      content: public class ReceiverDisconnectedException
  - uid: com.microsoft.azure.eventhubs._server_busy_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._server_busy_exception.yml
    name: ServerBusyException
    nameWithType: ServerBusyException
    fullName: com.microsoft.azure.eventhubs.ServerBusyException
    type: Class
    summary: |-
      <p>Server ausgelastet Ausnahme wird ausgelöst, wenn die aktuelle Entität Aktivität übermäßigen Last auf dem Dienst eingefügt wurde. Wenn diese Ausnahme festgestellt Benutzer sollte über mindestens 4 Sekunden erneut keine Wiederholung-Runtime-Vorgänge für die genannten Entität warten.</p>
      <p></p>
    syntax:
      content: public class ServerBusyException
  - uid: com.microsoft.azure.eventhubs._timeout_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._timeout_exception.yml
    name: TimeoutException
    nameWithType: TimeoutException
    fullName: com.microsoft.azure.eventhubs.TimeoutException
    type: Class
    summary: |-
      <p>Diese Ausnahme wird ausgelöst, wenn der Vorgang der zuvor festgelegten Zeitlimit überschritten hat. Benutzer sollte überprüfen, dass die Konnektivität zwischen Clientprozess und Dienst fehlerfrei ist.</p>
      <p></p>
    syntax:
      content: public class TimeoutException
  - uid: com.microsoft.azure.eventhubs._error_context
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._error_context.yml
    name: ErrorContext
    nameWithType: ErrorContext
    fullName: com.microsoft.azure.eventhubs.ErrorContext
    type: Class
    syntax:
      content: public class ErrorContext